YoCode/Output/WebWriter.cs

```diff
@@ -73,6 +73,7 @@ private string BuildReport()
         public void WriteReport()
         {
             File.WriteAllText(OUTPUT_PATH, BuildReport());
+            HtmlReportLauncher.LaunchReport("YoCodeReport.html");
```

Comment:
This function gets called a lot, isn't this going to make it pop up a lot of times?

Reply:
It will only be called when all report elements are added and report can be built. So it should only be called once per report. However, there was one issue with printing introduction, as PrintIntroduction() method called WriteReport(). It will be fixed in further commit